<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>892</width>
    <height>614</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>qrpmbuild-gui</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res.qrc">
    <normaloff>:/img/qrpmbuild-gui.png</normaloff>:/img/qrpmbuild-gui.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="2">
     <widget class="QPushButton" name="pushButton_removeFile">
      <property name="text">
       <string>Remove File</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="5">
     <widget class="QTableWidget" name="tableWidget_files">
      <property name="rowCount">
       <number>0</number>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>380</number>
      </attribute>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>File</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Install Path</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="0" column="0" colspan="5">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="8" column="4">
        <widget class="QLineEdit" name="lineEdit_packager">
         <property name="text">
          <string>xx &lt;xxx@mail.com&gt;</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1" colspan="4">
        <widget class="QTextEdit" name="textEdit_description">
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Hack';&quot;&gt;description&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item row="6" column="4">
        <widget class="QLineEdit" name="lineEdit_url">
         <property name="text">
          <string>https://xxx</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>description</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit_summary">
         <property name="text">
          <string>summary</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEdit_version">
         <property name="text">
          <string>1.0.0</string>
         </property>
        </widget>
       </item>
       <item row="8" column="2">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Packager</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="lineEdit_source">
         <property name="text">
          <string>https://xxx</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Release</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>License</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Group</string>
         </property>
        </widget>
       </item>
       <item row="2" column="4">
        <widget class="QLineEdit" name="lineEdit_release">
         <property name="text">
          <string>1.0</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Version</string>
         </property>
        </widget>
       </item>
       <item row="4" column="4">
        <widget class="QComboBox" name="comboBox_group">
         <item>
          <property name="text">
           <string extracomment="（娱乐/游戏）">Amusements/Games </string>
          </property>
         </item>
         <item>
          <property name="text">
           <string comment="（娱乐/图形）" extracomment="（娱乐/图形）">Amusements/Graphics</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string extracomment="（应用/文档）">Applications/Archiving </string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Applications/Communications</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Applications/Databases</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Applications/Editors</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Applications/Emulators</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Applications/Engineering</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Applications/File</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Applications/Internet </string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Applications/Multimedia</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Applications/Productivity</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Applications/Publishing</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Applications/System</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Applications/Text</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Development/Debuggers</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Development/Languages </string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Development/Libraries</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Development/System</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Development/Tools </string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Documentation</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>System Environment/Base</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>System Environment/Daemons</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>System Environment/Kernel </string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>System Environment/Libraries</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>System Environment/Shells </string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>User Interface/Desktops</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>User Interface/X </string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>User Interface/X Hardware Support </string>
          </property>
         </item>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Distribution</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="comboBox_license">
         <item>
          <property name="text">
           <string comment="自由软件" extracomment="自由软件">GPL</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>BSD</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>MIT</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string comment="公共域" extracomment="公共域">Public Domain</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string comment="贡献" extracomment="贡献">Distributable</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string comment="商业" extracomment="商业">commercial</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string comment="共享" extracomment="共享">Share</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Source</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Summary</string>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>URL</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QLineEdit" name="lineEdit_name">
         <property name="text">
          <string>name</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QLineEdit" name="lineEdit_Distribution">
         <property name="text">
          <string>Linux</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QPushButton" name="pushButton_generate">
      <property name="text">
       <string>Generate</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="pushButton_addFile">
      <property name="text">
       <string>Add file</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="pushButton_removeAll">
      <property name="text">
       <string>Remove All</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEdit_summary</tabstop>
  <tabstop>lineEdit_name</tabstop>
  <tabstop>lineEdit_version</tabstop>
  <tabstop>lineEdit_release</tabstop>
  <tabstop>lineEdit_source</tabstop>
  <tabstop>lineEdit_url</tabstop>
  <tabstop>lineEdit_Distribution</tabstop>
  <tabstop>lineEdit_packager</tabstop>
  <tabstop>textEdit_description</tabstop>
  <tabstop>pushButton_addFile</tabstop>
  <tabstop>pushButton_generate</tabstop>
 </tabstops>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
